index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
Please add on head meta:
-Title
-Keywords.
-Description.
-Favicon.

These changes are going to make the website more SEO friendly.

Reply:
♻️ Great job pointing out that adding meta tags would make the website more SEO friendly 😉😉. I'd suggest though, along with rephrasing your requirement as a more friendly suggestion, you dig a little bit deeper either by providing some examples or external documentation regarding those tags you are asking for. That would help the dev understand why it would make the page more SEO friendly and how to use them.